html/webappapis/dynamic-markup-insertion/opening-the-input-stream/no-new-global.window.js

```diff
@@ -0,0 +1,51 @@
+async_test(t => {
+  const frame = document.body.appendChild(document.createElement("iframe"));
+  t.add_cleanup(() => frame.remove());
+  frame.src = "resources/global-variables-frame.html";
+  frame.onload = t.step_func_done(() => {
+    assert_equals(frame.contentWindow.hey, "You", "precondition");
+    frame.contentDocument.open();
+    assert_equals(frame.contentWindow.hey, "You", "actual check");
+    // Ensure a load event gets dispatched to unblock testharness
```

Comment:
This test fails in Firefox, so this line wasn't reached and yet testharness wasn't blocked waiting for the load event. If this is just defensive coding, maybe put it in a cleanup step before the assert still?

Reply:
That's because of the cleanup step I added earlier in the test with `frame.remove()`. I could remove this.